packer/http/fedora-22/ks-ganeti.cfg

```diff
@@ -0,0 +1,46 @@
+install
+cdrom
+lang en_US.UTF-8
+keyboard us
+network --bootproto=dhcp
+rootpw osuadmin
+firewall --disabled
+authconfig --enableshadow --passalgo=sha512
+selinux --permissive
+timezone UTC
+bootloader --location=mbr
+text
+skipx
+zerombr
+clearpart --all --initlabel
+part /boot --fstype="ext4" --size=256
+part swap --fstype="swap" --size=512
+part / --fstype="ext4" --grow --size=100
+bootloader --location=mbr --append="norhgb biosdevname=0"
+auth  --useshadow  --enablemd5
+firstboot --disabled
+reboot
+user --name=osuadmin --plaintext --password osuadmin
+url --url=http://fedora.osuosl.org/linux/releases/22/Server/x86_64/os/
+repo --name=updates --baseurl=http://fedora.osuosl.org/linux/updates/22/x86_64/
+
+%packages --nobase --ignoremissing
+bzip2
+man
+man-pages
+net-tools
+nfs-utils
+rsync
+tar
+vim
+wget
+%end
+
+%post
+# update root certs
+wget -O/etc/pki/tls/certs/ca-bundle.crt http://curl.haxx.se/ca/cacert.pem
```

Comment:
Is there a reason to not grab this straight from mozilla?

Also, the certs in f22 should be quite up to date.